src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
```

Comment:
Consider adding a generic font family as a fallback for 'Avenir' to ensure proper rendering if the custom font fails to load. For example, `font-family: 'Avenir', sans-serif;`.